man/gbz80.7

```diff
@@ -754,22 +754,27 @@ Cycles: 1
 Bytes: 1
 .Pp
 Flags: None affected.
-.Ss JR e8
-Relative Jump by adding
-.Ar e8
-to the address of the instruction following the
-.Sy JR .
-To clarify, an operand of 0 is equivalent to no jumping.
+.Ss JR n16
+Relative Jump to address
+.Ar n16 .
+The address is encoded as a relative 8-bit offset from the instruction following the
```

Comment:
```suggestion
The address is encoded as a signed 8-bit offset from the start of the instruction following the
```

Reply:
Instead of "the start of the instruction following the JR", I'd use "the end of the JR instruction".